data/mods/Medieval_Stuff/shields.json

```diff
@@ -156,7 +156,7 @@
     "coverage": 30,
     "encumbrance": 8,
     "material_thickness": 3,
-    "techniques": [ "WBLOCK_2" ],
+    "techniques": [ "WBLOCK_4" ],
```

Comment:
Was mostly to be consistent with original behavior, where bucklers had 1 level higher than most shields. Should this stay at block 2 instead?